staging/benchmarks/json/003_benchmarks_hcqar_updates.json

```diff
@@ -0,0 +1,4388 @@
+[
```

Comment:
Do we need this file for any reason?

Reply:
Now that I understand better what's going on I think what should be changed is storing benchark csv's in the `/staging/benchmarks/csv` directory. I understand storing csv's in that directory was a convention from before this PR so does not need to be changed as part of this PR. However, wanted to mention that data in `/staging/*` is intended to be valid measures or benchmark json (or a valid subset of fields which are joinable on `measureId`) according the respective schemas and this data meant to be reserved for data that is used during the `build:measures` or `build:benchmarks` scripts.

The place for source data such as `staging/benchmarks/csv/003_benchmarks_hcqar_updates.csv` is actually `/util/benchmarks`. Which perhaps itself should be changed from `util` to `source_data` or just `data`

Let me know what you think.

Reply:
@abarciauskas-bgse so are you saying that _everything_ in `staging/benchmarks/csv/` should be moved over to `util/benchmarks/`? Or just the `003` file? And either way, the `util/benchmarks/` directory is kinda confusing... right now, we have a CSV, a JSON file, and a JS script in there. We could make a `csv` directory in there to put the stuff from `staging/benchmarks/csv/` into as well as the `2017.csv` in there, but not sure what to do with the `additional-benchmarks-2017.json` file...? Thoughts?

Reply:
I agree `/util/benchmarks/` is confusing, but think we should leave it as-is until we are prepared to implement a more encompassing solution.